### PR TITLE
bug: Fix errors inside comments breaking `Clion` error check

### PR DESCRIPTION
**Pull request by pmnoxx**
Clion shows errors inside code that's in doc strings. We should fix it.

### CODE REVIEW

File: core/metrics/src/lib.rs
Comment (mina86) [inline core/metrics/src/lib.rs:41]: This `lazy_static!` section should be removed now since the statics are no longer used. And we no longer need `extern crate lazy_static`.

Comment (pmnoxx) [reply core/metrics/src/lib.rs:41]: @mina86 Yes, I submitted. A week ago, still no reviews :/

https://github.com/near/nearcore/pull/5318
https://github.com/near/nearcore/pull/5366
https://github.com/near/nearcore/pull/5374

**Review by matklad — APPROVED** [review]
LGTM I guess, although I am wondering if the value brought by these doc comments outweight the maintainance burden?

**Comment by pmnoxx** [comment]
> LGTM I guess, although I am wondering if the value brought by these doc comments outweight the maintainance burden?

@matklad Sorry, a few days ago I added `S-automerge` flag to this PR. Your PR got automatically merged after the approval.
Next time, I'll not add `S-automerge` to changes, before they get approved, to avoid automerge from happening.

**Comment by pmnoxx** [comment]
> LGTM I guess, although I am wondering if the value brought by these doc comments outweight the maintainance burden?

I'm planning to remove this whole file after:
#5318 #5366 #5374

Though, meanwhile, I want to fix issues for CLion, no to bother people.

**Pull request merged**
